integrations/lib/multiwoven/integrations/destination/mailchimp/config/spec.json

```diff
@@ -0,0 +1,28 @@
+{
+  "documentation_url": "",
+  "stream_type": "static",
+  "connection_specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Mailchimp",
+    "type": "object",
+    "required": ["api_key", "list_id"],
+    "properties": {
+      "api_key": {
+        "type": "string",
+        "multiwoven_secret": true,
+        "title": "API Key",
+        "order": 0
+      },
+      "list_id": {
+        "type": "string",
+        "title": "List Id",
+        "order": 1
+      },
+      "email_template_id": {
```

Comment:
@xyfer17 what is email_template_id?
is it mandatory or specific to any stream?

Reply:
`email_template_id ` is the template id for the email campaigns, if user have email template already then it is used, as it's not mandatory.